src/commands/createSlot.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { SlotsTreeItem } from '../tree/SlotsTreeItem';
+import { SlotTreeItem } from '../tree/SlotTreeItem';
+
+export async function createSlot(context: IActionContext, node?: SlotsTreeItem, resourceGroup?: string): Promise<string> {
```

Comment:
Remove `resourceGroup` parameter. We don't support createSlot in [the api](https://github.com/microsoft/vscode-azurefunctions/wiki/Azure-Functions-API-(Preview)) so no need to add code that will never be used.